proto/cel/policy/policy.proto

```diff
@@ -0,0 +1,88 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package cel.policy;
+
+option cc_enable_arenas = true;
+option go_package = "cel.dev/policy";
+option java_multiple_files = true;
+option java_package = "dev.cel.policy";
+
+message PolicySpec {
+  enum Semantic {
+    FIRST_MATCH = 0;
```

Comment:
FIRST_MATCH is the current default, but since this is a proto enum -- we should probably have an `UNSPECIFIED` element as the 0th entry

Reply:
The current policy compiler defaults semantic even if you don't set it. So differentiating between set/unset is not super helpful currently. There is also the question of what should happen if it is unset? We can always add `optional` in-front to differentiate.

Reply:
Though I can see a place where we integrators can treat unset as whatever default they want. So lets do SEMANTIC_UNSPECIFIED.